kolibri/tasks/api.py

```diff
@@ -190,6 +184,27 @@ def localdrive(self, request):
 
         return Response(out)
 
+    @list_route(methods=['post'])
+    def channelinfo(self, request):
```

Comment:
Where is this endpoint mounted? What is the URL for it?

Reply:
Based on the [spec doc](https://docs.google.com/document/d/1FGR4XBEu7IbfoaEy-8xbhQx2PvIyxp0VugoPrMfo4R4/edit#heading=h.i5g7u0h0h7wn) it should be mounted to `/api/remotechannels`